components/02-molecules/accordion/_accordion-item.twig

```diff
@@ -0,0 +1,29 @@
+{% set accordion__item__base_class = 'accordion-item' %}
+
+{% set accordion__item__button__content %}
+  {{ accordion__item__heading }}
+  {% include "@atoms/images/icons/_icon.twig" with {
+    icon__name: 'angle-down',
+    icon__blockname: accordion__item__base_class,
```

Comment:
Add icon__decorative here to stop screen reader from saying "graphic".

Reply:
Done